src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -957,7 +932,7 @@ class CreateVmModal extends React.Component {
                 storageSize: convertToUnit(this.state.storageSize, this.state.storageSizeUnit, units.GiB),
                 storagePool: this.state.storagePool,
                 storageVolume: this.state.storageVolume,
-                unattended: this.state.unattendedInstallation,
+                unattended: unattendedInstallation, // TODO
```

Comment:
This still needs to be resolved?

Reply:
Ah no, that was from ancient times when I started this PR. Resolved already